packages/icons/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["esnext", "dom"],
+    "target": "es6",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "jsx": "react",
+    "skipLibCheck": true,
+    "noImplicitAny": true,
```

Comment:
you can also add here `allowJs: false` and convert the metadata file to `.ts`

Reply:
notice that since you're only including `src` folder in this file, so we won't need, yet, to change the rest of the scripts (though we should sometime soon)